app/pages/browse-solutions/template.njk

```diff
@@ -7,7 +7,7 @@
   <div class="nhsuk-width-container" data-test-id="browse-solutions">
     <div class="nhsuk-u-padding-top-5" data-test-id="go-back-link">
       {{ backLink({
-        "href": "/",
+        "href": "./" | addBaseUrlFilter,
```

Comment:
What would ./ with the addBaseUrlFilter applied look like? would it just be a . with the trailing slash stripped?

Reply:
yeah I'm seeing that now 😂